subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -20,45 +18,25 @@
 #include <bluetooth/gatt.h>
 #include <bluetooth/l2cap.h>
 
-#include "../host/conn_internal.h"  /* To avoid build errors on use of struct bt_conn" */
+#include "../../host/conn_internal.h"  /* To avoid build errors on use of struct bt_conn" */
```

Comment:
This is in order to log `conn` pointer values? If so, then perhaps it's just cleaner to `void *` cast them instead.

Reply:
That works